cookbooks/fb_networkmanager/metadata.rb

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2019-present, Vicarious, Inc.
+# Copyright (c) 2019-present, Facebook, Inc.
+name 'fb_networkmanager'
+maintainer 'Facebook'
+maintainer_email 'noreply@facebook.com'
+license 'Apache-2.0'
+source_url 'https://github.com/facebook/chef-cookbooks'
+description 'Installs/Configures Network Manager'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+# never EVER change this number, ever.
+version '0.1.0'
+supports 'ubuntu'
```

Comment:
This should work as-is on Debian too

Reply:
Sure. I suspect it works on Fedora as well. I'm hoping @michel-slm  will give it a spin. I already sent this to him.